### PR TITLE
DCS-1435 Update dependencies

### PR DESCRIPTION
**Pull request by andrewrlee**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,39 +52,40 @@
...
     "nyc": "15.1.0",
-    "prettier": "2.4.1",
+    "prettier": "2.5.1",
     "set-cookie-parser": "^2.4.8",
-    "supertest": "^6.1.6",
-    "ts-jest": "^27.0.7",
-    "typescript": "^4.5.2"
+    "supertest": "^6.2.2",
+    "ts-jest": "^27.1.3",
+    "typescript": "^4.5.5"
```
Comment (andrewrlee) [inline package.json:88]: there's some known issues with:
* redis
* MOJ frontend
* Govuk frontend


We'll need to create a new ticket for dealing with these

File: package.json [hunk 2]
```diff
@@ -10,41 +10,40 @@
...
-    "helmet": "^4.6.0",
-    "http-errors": "^1.8.1",
+    "helmet": "^5.0.2",
+    "http-errors": "^2.0.0",
     "jsonwebtoken": "^8.5.1",
     "jwt-decode": "^3.1.2",
     "moment": "^2.29.1",
     "moment-business-days": "^1.2.0",
     "nocache": "^3.0.1",
-    "nock": "^13.2.1",
```
Comment (andrewrlee) [inline package.json:39]: nock shouldnt be a dev dependency

File: backend/jobs/eventsRetriever.ts [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import { Moment } from 'moment'
-import parse from 'csv-parse'
+import { parse } from 'csv-parse'
```
Comment (andrewrlee) [inline backend/jobs/eventsRetriever.ts:2]: api change

**Review by GurnankCheema — APPROVED** [review]
LGTM

**Pull request merged**